util/topgen/templates/toplevel_pkg.sv.tpl

```diff
@@ -37,6 +37,26 @@ package top_${top["name"]}_pkg;
    * Memory size for ${name} in top ${top["name"]}.
    */
   parameter int unsigned ${region.size_bytes_name().as_c_define()} = ${hex_size_bytes};
+  ## TODO: we need a more holistic approach to declare memories and IPs sitting in the
```

Comment:
Please create an issue and reference it here (`TODO(#....)`) so we can track this TODO item

Reply:
Issue: #25313

Reply:
I'll link to that in a follow up PR